### ISSUE TITLE
--debug dose not work anymore after node v8

### ISSUE THREAD

**Issue by mlibre** [remy/nodemon#1062]
$ node -v
v8.1.2
$ nodemon --debug app.js 
[nodemon] 1.11.0
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: *.*
[nodemon] starting `node --debug app.js`
(node:9220) [DEP0062] DeprecationWarning: `node --debug` and `node --debug-brk` are invalid. Please use `node --inspect` or `node --inspect-brk` instead.
[nodemon] app crashed - waiting for file changes before starting...
$ nodemon app.js 
[nodemon] 1.11.0
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: *.*
[nodemon] starting `node app.js`
#Server. Running On 3000 and So on ........................

**Comment by remy** [comment]
Use `--inspect` as per the message from node.

```
$ nodemon app.js -- --inspect
````

**Comment by Jokero** [comment]
should be `nodemon -- --inspect app.js`

**Comment by Suzal3579** [comment]
Neither `nodemon -- --inspect app.js `  nor `nodemon app.js -- --inspect` worked for me. Any alternatives?

**Comment by Jokero** [comment]
Check `node --inspect app.js` directly. If it does not work, problem is not related to nodemon. If you run app inside docker container and want to have remote debugging, specify `0.0.0.0:9229` explicitly:
```sh
nodemon -- --inspect 0.0.0.0:9229 app.js
```
Last one works for me

**Comment by remy** [comment]
Damnit - latest nodemon isn't passing this argument. I'll raise a new issue as it's unrelated to this initial issue.

**Comment by Jokero** [comment]
Yes, I use not newest version. Then `inspect-brk` maybe also doesn't work

**Comment by remy** [comment]
Fixed and live in nodemon@1.12.7

**Comment by RooSoft** [comment]
Using 1.12.7, the `--inspect` switch is still not working....

**Comment by Suzal3579** [comment]
@remy thank you, works great now 👍

**Comment by remy** [comment]
@roosoft please file a new issue with full details and if possible a PR with resolution.

**Comment by mstellaluna** [comment]
I am currently having this issue also, inspect is loading but its not stopping at the debugger; specified in one of my required files that my main app.js file reads from

**Comment by remy** [comment]
@mstellaluna you've commented on a closed issue, but I've just tried the same thing and working…but you've not really included much information to replicate fully.

Here's windows 10 for me:

![nodemon-inspect](https://user-images.githubusercontent.com/13700/34049220-7fbca02c-e1ae-11e7-9e52-1db5de54bc87.PNG)

**Comment by mstellaluna** [comment]
@remy sorry I apologize I didn't notice it was closed as other were commenting as of recent.

**Comment by vitalii-svitla** [comment]
Is it possible to attach a Vs Code debugger (i'm following https://blog.docker.com/2016/07/live-debugging-docker/) to :9229? Its failing to connect in my case.

**Comment by vitalii-svitla** [comment]
Found updated version here: https://github.com/docker/labs/blob/master/developer-tools/nodejs-debugging/VSCode-README.md
Seems to work for me.
